utility/aws.py

```diff
@@ -0,0 +1,148 @@
+import logging
+import time
+
+import boto3
+
+
+logger = logging.getLogger(name=__file__)
+
+
+class AWSTimeoutException(Exception):
+    pass
+
+
+class AWS(object):
+    """
+    This is wrapper class for AWS
+    """
+
+    _ec2_client = None
+    _ec2_resource = None
+
+    @property
+    def ec2_client(self):
+        """ Property for ec2 client
+
+        Returns:
+            boto3.client: instance of ec2
+        """
+        if not self._ec2_client:
+            self._ec2_client = boto3.client('ec2')
```

Comment:
When I was directly invoking the script via gitlab, I noticed that it asked me for region to be set either in .aws/env/or in this call. I think we can set it here with default region.

Reply:
didn't hit such issue, probably some miss configured AWS in your home? We can add one in defaults and read it from there.

Reply:
Changed logic to have default region taken from default or config, check commit:  https://github.com/red-hat-storage/ocs-ci/pull/23/commits/6659aa4f0c41c1972ffa9f7306ae8b7a8f4f52c1